src/main/java/gov/cabinetoffice/gap/adminbackend/services/SchemeEditorService.java

```diff
@@ -81,5 +87,30 @@ private List<String> getEmailsFromUserSubBatch(final List<String> userSubs, fina
                 awsEncryptionService.decryptField(item.emailAddress())).toList();
     }
 
+    public SchemeEntity addEditorToScheme(Integer schemeId, String editorEmailAddress, String jwt) {
+        AdminSession session = HelperUtils.getAdminSessionForAuthenticatedUser();
+        SchemeEntity scheme = this.schemeRepo.findById(schemeId).orElseThrow(EntityNotFoundException::new);
+        List<GrantAdmin> existingEditors = scheme.getGrantAdmins();
+        GrantAdmin editorToAdd;
+        try {
+            editorToAdd = userService.getGrantAdminIdFromUserServiceEmail(editorEmailAddress, jwt);
+        } catch (Exception e) {
```

Comment:
Is there a more specific exception/group of exceptions we could catch here? Its not great practice to just blanket catch exceptions this way in Java

Reply:
It's been a minute so will have a look if there's something better to catch here.